website/scripts/generate_sitemap.js

```diff
@@ -0,0 +1,46 @@
+const fs = require('fs')
+const globby = require('globby')
+const prettier = require('prettier')
+
+/** Based on this blog by Lee Robinson: https://leerob.io/blog/nextjs-sitemap-robots */
+
+;(async () => {
+  const prettierConfig = await prettier.resolveConfig('./.prettierrc.js')
+
+  // Ignore Next.js specific files (e.g., _app.js) and API routes.
+  const pages = await globby([
+    'pages/**/*{.js,.tsx,.jsx}',
+    //  'content/**/*.mdx', // docs routes
```

Comment:
Only printing out the static routes currently as running the a11y test on all docs routes would take a really long time. Obviously the sitemap should have all routes so maybe I shouldn't use that as the base for a11y testing.